src/_guides/libraries/objective-c-interop.md

```diff
@@ -328,6 +341,34 @@ to the correct isolate.
 For an example of this,
 see the implementation of [`package:cupertino_http`][].
 
+There are a few steps required to handle callbacks:
```

Comment:
@liamappelbe can comment more on timeline, but I don't think we have an exact timeline at the moment. I do think it will be at least a few months. 

The demo we created for Flutter Forward is pretty simple so maybe it makes sense to hold off until we've published that demo and then use it as a code sample (I don't think it will be updated so maintenance is less of an issue). What do you think?

Reply:
If it's a temporary solution / experimental package, is there a way to create a page that makes that clear? Maybe it makes more sense to have a Medium blog post or something and then just link to it in this page?

Reply:
Yeah, it's definitely going to be O(months). I think making it a separate page and using inline snippets from the flutter forward demo, rather than linking to cupertino_http makes the most sense.

Reply:
Sounds like we all agree on the separate page then. And if there's a demo with snippets, that would be fantastic to use instead :)